PWGHF/TableProducer/candidateSelectorD0.cxx

```diff
@@ -263,7 +268,7 @@ struct HfCandidateSelectorD0 {
       if (!(candidate.hfflag() & 1 << aod::hf_cand_2prong::DecayType::D0ToPiK)) {
         hfSelD0Candidate(statusD0, statusD0bar, statusHFFlag, statusTopol, statusCand, statusPID);
         if (applyMl) {
-          hfMlD0Candidate(outputMl);
+          hfMlD0Candidate(outputMlNotPreselectedD0, outputMlNotPreselectedD0bar);
```

Comment:
Before this, e.g. in line `267` you should add
```
outputMlD0.clear();
outputMlD0bar.clear();
```
I cannot put it as a suggestion because that line is not modifed in your PR.